src/BTCPayServer.Lightning.Common/CreateInvoiceParams.cs

```diff
@@ -52,6 +52,8 @@ public uint256 DescriptionHash
             }
         }
         public bool DescriptionHashOnly { get; set; }
+        
+        [JsonConverter(typeof(TimeSpanJsonConverter.Seconds))]
```

Comment:
`CreateInvoiceParams` isn't meant to be serialized. You shouldn't put this here.